errtracker/errtracker.go

```diff
@@ -119,6 +130,8 @@ func Report(snap, errMsg, dupSig string, extra map[string]string) (string, error
 		"CoreSnapdBuildID":   coreBuildID,
 		"Date":               timeNow().Format(time.ANSIC),
 		"Snap":               snap,
+		"SnapConfineCurrent": snapConfineProfileDigest(""),
```

Comment:
Done, I kept it brief for golang reflow reasons but I called it this way before :)